cypress/e2e/dashboard/sponsors.cy.js

```diff
@@ -0,0 +1,69 @@
+import { expectToBeRejected } from '../../support/util';
+
+const testSponsor = {
+  name: 'Test Sponsor',
+  website: 'http://example.com',
+  logo_path: 'logo.png',
+  type: 'OTHER',
+};
+
+describe('sponsors dashboard', () => {
+  beforeEach(() => {
+    cy.exec('npm run db:seed');
+  });
+
+  it('lets an instance owner create sponsors', () => {
+    cy.login();
+    cy.visit('/dashboard/sponsors/new');
+
+    cy.findByRole('textbox', { name: 'Sponsor Name' }).type(testSponsor.name);
+    cy.findByRole('textbox', { name: 'Website Url' }).type(testSponsor.website);
+    cy.findByRole('textbox', { name: 'Logo Path' }).type(testSponsor.logo_path);
+    cy.findByRole('combobox', { name: 'Sponsor Type' }).select(
+      testSponsor.type,
+    );
+    cy.findByRole('button', { name: 'Add New Sponsor' }).click();
+
+    cy.findByRole('heading', { name: 'Sponsors' });
+    cy.findByRole('link', { name: testSponsor.name }).click();
+    cy.get('[data-cy=name]').contains(testSponsor.name);
+    cy.get('[data-cy=website]').contains(testSponsor.website);
+    cy.get('[data-cy=type]').contains(testSponsor.type);
+  });
+
+  it('lets an instance owner edit sponsors', () => {
+    cy.login();
+
+    cy.visit('/dashboard/sponsors/1/edit');
+    cy.findByRole('textbox', { name: 'Sponsor Name' })
+      .clear()
+      .type(testSponsor.name);
+    cy.findByRole('textbox', { name: 'Website Url' })
+      .clear()
+      .type(testSponsor.website);
+    cy.findByRole('textbox', { name: 'Logo Path' })
+      .clear()
+      .type(testSponsor.logo_path);
+    cy.findByRole('combobox', { name: 'Sponsor Type' }).select(
+      testSponsor.type,
+    );
+
+    cy.findByRole('button', { name: 'Save Sponsor Changes' }).click();
+    cy.findByRole('heading', { name: 'Sponsors' });
+    cy.findByRole('link', { name: testSponsor.name }).click();
+    cy.get('[data-cy=name]').contains(testSponsor.name);
+    cy.get('[data-cy=website]').contains(testSponsor.website);
+    cy.get('[data-cy=type]').contains(testSponsor.type);
+  });
+
+  it('prevents members from managing sponsors', () => {
```

Comment:
```suggestion
  it('prevents chapter admins from managing sponsors', () => {
```
We might as well test against admins. If they can't do it, members should also be prevented.